tests/util.py

```diff
@@ -1,8 +1,9 @@
 import os
 import time
+from functools import partial
 
 import pytest
-from rest_tools.client import OpenIDRestClient
```

Comment:
Is OpenIDRestClient still used anywhere, in any repo?

Reply:
Not in this repo, but it should be used for new Keycloak clients.  The change was for using service accounts with refresh tokens.  Actual user accounts work fine, which is where this would come in.